content/04-postgresql/12-short-guides/02-creating-deleting-users.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'How to use create and delete users in PostgreSQL'
+metaTitle: "How to use create and delete users in PostgreSQL"
+metaDescription: "This short guide will cover how to create and delete users in PostgreSQL."
+---
+
+## Introduction
+
+More often than not, when working with a database you will need to be able to create and delete users. Whether working in a team that has multiple members needing access to the database or creating and deleting test users to simulate database privileges, knowing how to add and remove database users is important.
+
+In this short guide, we will cover the basics of creating and deleting database users in PostgreSQL. These commands will have you ready to get started on granting database access and establish a foundation for future [role management](https://www.prisma.io/dataguide/postgresql/authentication-and-authorization/role-management).
+
+
+## Create a user
+
+PostgreSQL has the `CREATE USER` psql command and `createuser` for the command line. Both options define a new database user account. In order to create users, you need to be a [superuser role](https://www.prisma.io/dataguide/postgresql/authentication-and-authorization/role-management#what-is-a-superuser-role) for the database cluster.
+
+The basic syntax for `createuser` looks like this:
+
+```
+createuser <options>
+```
+and for `CREATE USER`:
+
+```
+CREATE USER <name><options>
+```
+The [options component](https://www.postgresql.org/docs/current/app-createuser.html) of the command is where you can define parameters such as username, whether this user can create databases, superuser, and the need for a password when connecting to the server. 
+
+The syntax can look something like:
+
+```
+createuser --createdb --password user1
+```
+In addition to creating users, it is important to know the different privileges that can be granted so users are only able to work with the database in specified ways. For the purpose of this guide we are just covering the basics to get started, but you can read more about [granting privleges in PostgreSQL](https://www.prisma.io/dataguide/postgresql/authentication-and-authorization/managing-privileges) for more advanced settings.
+
+## Delete a user
+
+With the ability now to create users, it is important to also know how to delete a user when their access is no longer needed. Similar to creating a user, there are two ways to delete one. You can use both `dropuser` and `DROP USER`.
```

Comment:
I'd reiterate here that `dropuser` is a command line utility and that `DROP USER` is a `psql` command just to avoid confusion.